src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Net.Http.WinHttpHandler</AssemblyName>
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <!-- Although we have a NS configuration, the actual UWP implementation is a shim over WinRT: so just validate against OneCore -->
```

Comment:
Here is what I think
1. If the package previously shipped a netstandard ref, we shouldn't break that.  It needs to continue to support reference from past frameworks it supported.
 -  If the reason for jagged support is only missing PInvokes from the WACK, then its fine to have a windows-netstandard implementation.  If folks want to experiment with it they can.  If there is a good case for using this in UWP then we ask Windows to make those APIs available.  When in this mode we need to be careful that the library remains unreferenced by other libraries on that platform: we don't want to accidentally make some other library "experimental" because one of its dependencies happened to use an API outside the WACK.  We should also ensure that folks don't get in this mode by default, the closure of the UWP meta-package cannot have libraries that fail the WACK.
 - If the reason for jagged support is missing framework API (EG: Ref.Emit, Runtime.Loader, etc) then we shouldn't have a windows-netstandard implementation.  The library should ensure it cross-compiles for only the platforms which support the API it requires.
2. If its a brand new package and cannot support all platforms, opt for platform-specific refs and libs.  Customer scenarios need to make a case for making a netstandard ref and having land-mines.  Usually this will mean we didn't factor the API correctly which is something that can be corrected if it's the first release of a new library.